cfme/tests/infrastructure/test_resources.py

```diff
@@ -14,24 +14,17 @@ def pytest_generate_tests(metafunc):
                         scope="module")
 
 
-def get_integer_value(x):
-    return int(x.split(' ')[0])
-
-
 @pytest.mark.usefixtures("setup_provider_modscope")
 def test_number_of_cpu(provider, soft_assert):
-    soft_assert(get_integer_value(
-        provider.summary.properties.aggregate_node_cpu_resources.value) > 0,
-        "Aggregate Node CPU Resources is 0")
-    soft_assert(int(
-        provider.summary.properties.aggregate_node_cpus.value) > 0,
-        "Aggregate Node CPU is 0")
-    soft_assert(int(
-        provider.summary.properties.aggregate_node_cpu_cores.value) > 0,
-        "Aggregate Node CPU Cores is 0")
+    soft_assert((provider.summary.properties.aggregate_node_cpu_resources
+                .value.number) > 0, "Aggregate Node CPU Resources is 0")
```

Comment:
Why `.value.number`? Value is already an integer if it is parseable to an integer ...